api/src/main/java/net/kyori/adventure/text/format/StyleImpl.java

```diff
@@ -237,11 +198,11 @@ public boolean isEmpty() {
   public @NotNull Stream<? extends ExaminableProperty> examinableProperties() {
     return Stream.of(
       ExaminableProperty.of("color", this.color),
-      ExaminableProperty.of("obfuscated", this.obfuscated),
-      ExaminableProperty.of("bold", this.bold),
-      ExaminableProperty.of("strikethrough", this.strikethrough),
-      ExaminableProperty.of("underlined", this.underlined),
-      ExaminableProperty.of("italic", this.italic),
+      ExaminableProperty.of("obfuscated", this.decorations.get(TextDecoration.OBFUSCATED)),
+      ExaminableProperty.of("bold", this.decorations.get(TextDecoration.BOLD)),
+      ExaminableProperty.of("strikethrough", this.decorations.get(TextDecoration.STRIKETHROUGH)),
+      ExaminableProperty.of("underlined", this.decorations.get(TextDecoration.UNDERLINED)),
+      ExaminableProperty.of("italic", this.decorations.get(TextDecoration.ITALIC)),
```

Comment:
I wonder if we'd be better off making the map examinable and do it dynamically rather than this.

Reply:
Easy to do, will fix in the next commit!

Reply:
e285a7ec967d0890c07a0e015a83dbb0e408124d